### PR TITLE
Bundle of Bugfixes 

### PR DESCRIPTION
**Pull request by L74814**
## About The Pull Request

List of things that were fixed/changed:

1. Serbian Magboots have a button now for being turned on/off, and they start off now. Turning these through the button fixes the bug that make Serb magboots never trigger the effects of nonslip+magboots, so was like having only a fancy jackboot.
2. Fixes a lot of typos for consistency in the autolathe,  as for example: Wintermute being called a BR .20, Zoric called a FS SMG though it was a SA SMG. .20 Rubber Ammoboxes labeled as Practice.
3. Fixes IH Disks having the IH Breaching Hammer sprite.
4. Fixes the Invisible Box for Genetic Disks.
5. Deletes the parapen from Guild Contraband Crates.
5. Fixes carrion sprite not updating after reg.stasis (you could have regenerated a limb, but the sprite never updated properly). Still looking around about the carrion permanent paralysis, but that wasn't adressed for now.
6. Fixes Science Goggles having a weird sprite (black gloggles) when off. Consistency.
7. Deletes the Scanner part from Carrion. This change is because it was mechanically changed, but the description still reflected that.
8. Added spawn_blacklisted to: Fractal Integrated Circuit Disk (Debug Item) and the .70 Magazine.
9. The Old Knife oddity now uses a knife inhand sprite. Consistency, afterall it worked properly as a knife.
10. Deleted repeated text on selfload.dm
11. Fixes incendiary crates spawning with implantcases. Now they spawn with Incendiary Grenades properly.
12. Fixed Guild Artist SecHUD
13. Fixed Access Console Unable to assign Guild Artist Role
14. Fixes Self-CPR
15. Fixes RCON Input load values not working.
16. Gives TE access to the NTNET Program (which spawned with the program already) AND makes MEO spawn with the NTNET Program (which already had access)
17. Your race doesn't appear anymore when examining, as a petition of a jannie.

## Why It's Good For The Game

mmmm.... yes.... hmm.... less bugs.... 🐒 
## Changelog
:cl: HerBaon
fix: Exultant has NTNET Access now, and MEO spawns with the NTNET Program in their laptop.
fix: IH Hammer no longer appears when holding IH Disks.
fix: fixed a lot of bugs and typos around.
fix: Serbian Magboots work properly now.
/:cl:

### CODE REVIEW

File: code/datums/autolathe/guns.dm [hunk 1]
```diff
@@ -131,14 +131,18 @@
 	build_path = /obj/item/weapon/gun/projectile/automatic/z8
 
 /datum/design/autolathe/gun/wintermute
-	name = "FS BR .20 \"Wintermute\""
+	name = "FS AR .20 \"Wintermute\""
 	build_path = /obj/item/weapon/gun/projectile/automatic/wintermute
 
+/datum/design/autolathe/gun/kovacs
```
Comment (L74814) [inline code/datums/autolathe/guns.dm:137]: I moved this from the .30 section for consistency. I forgot to add the "SA BR .20" shit.

**Comment by L74814** [comment]
probably tomorrow i'l add more.

**Comment by L74814** [comment]
errr, probably this is fine for now. Other bugfixes are going into other PR.

**Comment by Firefox13** [comment]
Name your PRs properly please

**Comment by L74814** [comment]
1984

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**